ote_sdk/ote_sdk/utils/labels_utils.py

```diff
@@ -24,26 +23,3 @@ def get_empty_label(label_schema: LabelSchemaEntity) -> Optional[LabelEntity]:
     if empty_candidates:
         return empty_candidates[0]
     return None
-
-
-def get_leaf_labels(label_schema: LabelSchemaEntity) -> List[LabelEntity]:
```

Comment:
why?

Reply:
This functionality was replaced by label resolver